appcoins/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+  implementation fileTree(dir: 'libs', include: ['*.jar'])
+  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+  //converters
+  kapt("com.squareup.moshi:moshi-kotlin-codegen:1.9.3")
+  implementation("com.squareup.moshi:moshi-kotlin:1.9.3")
+  implementation("com.squareup.retrofit2:converter-moshi:2.9.0")
+
+  //network
+  implementation("com.squareup.retrofit2:retrofit:2.9.0")
+
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.7")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.7")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-rx2:1.3.7")
+  implementation("io.reactivex.rxjava2:rxjava:2.2.19")
```

Comment:
we could potentially remove this dependency if we only convert coroutines -> rx in client modules.

Reply:
We agreed to leave this like this so that when we need to change we will only have one place to change